components/server/src/prometheus-metrics.ts

```diff
@@ -286,3 +289,22 @@ export const jobsDurationSeconds = new prometheusClient.Histogram({
     buckets: [10, 30, 60, 2 * 60, 3 * 60, 5 * 60, 10 * 60, 15 * 60],
     labelNames: ["name"],
 });
+
+export const cacheHitsCounter = new prometheusClient.Counter({
+    name: "gitpod_server_cache_hits_total",
```

Comment:
This ideally shouldn't be `server` specific, as we should share the same metric from other systems. Also, I think we should model it as:
* Total requests (with label on hit/miss) to keep it semantic

`gitpod_redis_domain_cache_requests_total`